pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "squiral"
-version = "0.1.1"
```

Comment:
Well, it's already been 0.1.2
https://github.com/sadikkuzu/squiral/blob/main/pyproject.toml#L3